CMakeLists.txt

```diff
@@ -79,6 +79,18 @@ if (ENABLE_BUNDLED_DOCS)
     add_subdirectory(doc)
 endif(ENABLE_BUNDLED_DOCS)
 
+option(ENABLE_ASAN "Enable AddressSanitizer, a fast memory error detector based on compiler instrumentation" OFF)
+if (ENABLE_ASAN)
+    if (CMAKE_COMPILER_IS_GNUCC)
+        message(FATAL_ERROR
+            "\n"
+            " Tarantool-c does not support GCC's AddressSanitizer, use clang\n"
+            )
+    endif()
+
+    set(CMAKE_C_FLAGS  "${CMAKE_C_FLAGS} -fsanitize=address -fsanitize=leak")
+endif(ENABLE_ASAN)
+
```

Comment:
Doesn't gcc support address sanitizer nowadays? On my system (gcc version 9.3.0 (Ubuntu 9.3.0-17ubuntu1~20.04)), according to the manual, it does.

Reply:
GCC support address sanitizer nowadays, this part is same as in tarantool. Ok i fixed it